### PR TITLE
MYTHOS-437 - Major refactor to support single event stream.

### PR DESCRIPTION
**Pull request by kwang-mythical**
https://github.com/MythicalGames/saga-sdk-java/pull/25

### CODE REVIEW

File: streams/stream.proto [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import "streams/order/definition.proto";
+import "streams/user/definition.proto";
+
+option java_multiple_files = true;
+option java_package = "games.mythical.saga.sdk.proto.streams";
+
+package saga.rpc.streams;
+
+message StatusUpdate {
+    string trace_id = 1;
```
Comment (DustinLiang) [inline streams/stream.proto:21]: trace_id being defined here, can it be removed from each of the individual StatusUpdates?

**Review by DustinLiang — APPROVED** [review]
Looks good! Nice consolidation of the streams.

File: streams/stream.proto (hunk 1 above)
Comment (kwang-mythical) [reply streams/stream.proto:21]: ah yes, we'll include that in next refactor along with removing all mention of title_id

Comment (kwang-mythical) [reply streams/stream.proto:21]: cc: @erik-mythical don't know if you have already started with that or not.

**Pull request merged**